pkgs/development/python-modules/luna-soc/default.nix

```diff
@@ -13,15 +13,15 @@
 
 buildPythonPackage rec {
   pname = "luna-soc";
-  version = "0.2.2";
+  version = "0.3.2";
   pyproject = true;
-  disabled = pythonOlder "3.8";
+  disabled = pythonOlder "3.9";
```

Comment:
```suggestion
```

We don't support anything older than 3.11, can't we then also remove those conditionals completely?

Reply:
Fine by me, I just had these to be in line with the actual requirements of lib.